### PR TITLE
view-testimonials

### PR DESCRIPTION
**Pull request by hiixdayah**
## Description:
- View testimonials by sensei 
## Checklist:
- [x] Merged latest develop
- [x] PR title makes sense

### CODE REVIEW

File: src/services/mentorship.service.ts [hunk 1]
```diff
@@ -668,6 +668,25 @@ export default class MentorshipService {
...
+        accountId,
+      },
+    });
+
+    const mentorshipListingIds = mentorshipListings.map(
+      (ml) => ml.mentorshipListingId
+    );
+
+    return await Testimonial.findAll({
+      where: { mentorshipListingId: { [Op.in]: [...mentorshipListingIds] } },
```
Comment (ianleck) [inline src/services/mentorship.service.ts]:
```suggestion
      where: { mentorshipListingId: { [Op.in]: mentorshipListingIds } },
```

File: src/routes/mentorship.route.ts [hunk 2]
```diff
@@ -185,6 +186,15 @@ router.get(
   Utility.asyncHandler(MentorshipController.getTestimonialsByFilter)
 );
 
+//View List of Testimonials For ONE sensei
+router.get(
+  '/testimonial/list/:accountId',
+  passport.authenticate('isAuthenticated', { session: false }),
+  requireSameUser,
```
Comment (ianleck) [inline src/routes/mentorship.route.ts:193]: admin can't get all testimonials of sensei? ok or maybe no real need hm

**Review by ianleck — CHANGES_REQUESTED** [review]
admin should b

File: src/routes/mentorship.route.ts (hunk 2 above)
Comment (natalieloke) [reply src/routes/mentorship.route.ts:193]: this endpoint is for sensei to view all testimonials written by himself/herself

Comment (natalieloke) [reply src/routes/mentorship.route.ts:193]: Screenshot 2021-04-11 at 4 06 33 PM

Comment (natalieloke) [reply src/routes/mentorship.route.ts:193]: currently there's no use case regarding testimonials for admin

Comment (hiixdayah) [reply src/routes/mentorship.route.ts:193]: errr ya i didn't really see a need to so i didn't include

**Review by ianleck — APPROVED** [review]

**Pull request merged**